fhir-parent/pom.xml

```diff
@@ -642,7 +641,7 @@
                                     </tag>
                                 </tags>
                                 <additionalparam>-Xdoclint:none</additionalparam>
-                                <bottom><![CDATA[Copyright 2019, <a href="http://www.ibm.com">IBM Corporation<a>]]></bottom>
+                                <bottom><![CDATA[Copyright 2019, <a href="http://www.ibm.com">IBM Corporation<a><br></br>FHIR&reg; is the registered trademark of HL7 and is used with the permission of HL7.]]></bottom>
```

Comment:
super minor, but `<br></br>` could just be `<br />`

Reply:
same reasoning as above